_posts/en/newsletters/2020-10-21-newsletter.md

```diff
@@ -0,0 +1,183 @@
+  used in the private regression test mode (regtest); they are not
+  enforced in mainnet, testnet, or the default signet.  The plan is to
+  release the code in Bitcoin Core 0.21.0 and then prepare a subsequent
+  release (e.g. 0.21.1) that can begin enforcing the soft fork's new
+  rules when an activation signal is detected.  The particular
+  activation signal or signals to use are still being discussed in the
+  [##taproot-activation][] IRC chatroom ([logs][##taproot-activation
+  logs]).
+
+    {% comment %}<!-- $ git diff 450d2b2371...865d2c37e2 | grep '^+' \
+                        | grep -v '^+$' | grep -v '^+ */[/*]' | grep -v '^+ *\*' \
+                        | grep -v '^+++' | wc -l
+                      512
+    -->{% endcomment %}
+
+    The code [consists][sipa summary] of about [700 lines][sipa
+    consensus] of consensus-related changes (500 excluding comments and
+    whitespace) and [2,100 lines][sipa tests] of tests.  Over 30 people
+    directly reviewed this PR and its [predecessor][Bitcoin Core
+    #17977], and many others participated in developing and reviewing
+    the underlying research, the BIPs, the related code in libsecp256k1,
+    and other parts of the system.  There is still more work to do, but
+    getting the code properly reviewed was arguably the most critical
+    step---so we extend our most grateful appreciation to everyone who
+    helped achieve this milestone.
+
+- [Bitcoin Core #19988][] refactors the logic Bitcoin Core uses to
+  request transactions from its peers.  The major changes described by
+  the PR description include: Bitcoin Core will now more strongly prefer
+  requesting transactions from *outbound peers*---peers where the
+  connection was opened by the local node; Bitcoin Core will now request
+  an unlimited number of transactions from a peer rather than stopping
+  at 100 (though slow responders won't be preferred); and Bitcoin Core
+  won't keep track of as many pending transaction requests as before,
+  the previous limit being seen as excessive.  The change should also
+  make it easier to test that the transaction request logic is working
+  as expected, with multiple tests for that included in the merged PR.
+  This may help reduce the chance that a flaw can be exploited against
+  users of time-sensitive transactions (such as in LN and many other
+  contract protocols).
+
+- [Bitcoin Core #19077][] wallet: Add sqlite as an alternative wallet database and use it for new descriptor wallets FIXME:dongcarl
+
+- [Bitcoin Core #19770][] RPC: getpeerinfo: Deprecate "whitelisted" field (replaced by "permissions") FIXME:bitschmidty
+
+- [Bitcoin Core #17428][] periodically writes a file with the network
+  addresses of the node's two outbound block-relay-only peers.  The next
+  time the node starts, it reads this file and attempts to reconnect to
+  those same two peers.  This prevents an attacker with the ability to
```

Comment:
This isn't true. We only write the anchor connections file on a clean shutdown, so if an attacker can make us crash, then we won't persist connections.